api/src/controllers/users.js

```diff
@@ -56,12 +56,17 @@ const getInfoUserCtx = req => {
     throw { code: 400, reason: 'Missing required query params: role and/or facility_id' };
   }
 
-  if (!auth.isOffline([ params.role ])) {
+  try {
+    params.role = JSON.parse(params.role);
```

Comment:
How did you decide to use JSON here? Did you consider using something like a delimited string instead? 

Seems like using JSON opens up input types which we don't necessarily want to care about - like passing in an object. If the url is crafted manually, it also adds unnecessary syntax `a,b,` vs `["a","b"]`. Delimited strings seem like they would be both easier in code and easier to use from a user perspective.

Reply:
All CouchDB endpoints that use string parameters use JSON encoding, so it fits our stack. 
Would you greatly prefer a comma separated approach?

Reply:
No, that is a decent explanation. Would you add some type checking though, so we only ever pass around arrays of strings?